packages/web-twig/src/Resources/components/FileUploader/FileUploader.stories.twig

```diff
@@ -0,0 +1,20 @@
+{% extends 'layout/default.html.twig' %}
+
+{% block content %}
+    <section class="docs-Section">
```

Comment:
Why is the section not part of the include?

Reply:
Missed that!